google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableInstanceAdminStubSettings.java

```diff
@@ -516,15 +514,37 @@ public String extractNextToken(ListAppProfilesResponse payload) {
       RetrySettings settings = null;
       settings =
           RetrySettings.newBuilder()
-              .setInitialRetryDelay(Duration.ofMillis(5L))
+              .setInitialRetryDelay(Duration.ofMillis(1000L))
```

Comment:
Duration.ofSeconds(1) is easier to read

Reply:
This code is autogenerated by gapic-generator. So it would have to change in its templates